css/main.css

```diff
@@ -37,6 +37,10 @@ body {
   color: rgb(254,246,234);
   color: rgba(254,246,234,.8);
 }
+.footmrgn {
+  margin-right:5px;
+  margin-bottom:5px;
```

Comment:
Yeah the same thing goes for other classes like `.blip` etc. If they are re-used multiple times on a page then I'd say it is a good time to make a class for it, but for one-off things I'd just target the element.